packages/public/frontend/package.json

```diff
@@ -14,7 +14,9 @@
     "start": "next",
     "start-storybook": "start-storybook -p 9000 -c .storybook -s ./public"
   },
-  "dependencies": {},
+  "dependencies": {
+    "exenv": "^1.2.2"
```

Comment:
please move to devDevs

Reply:
And please change that to `^1.0.0` (except if there's a special reason that this has to be 1.2.2 or newer)

Reply:
done